src/main/java/org/jenkinsci/plugins/ghprb/GhprbPullRequest.java

```diff
@@ -207,11 +208,22 @@ private void checkWhiteListLabels() {
 
     private void checkSkipBuild() {
         synchronized (this) {
-            String skipBuildPhrase = helper.checkSkipBuild(this.pr);
+            String skipBuildPhrase = helper.checkSkipBuildPhrase(this.pr);
             if (!StringUtils.isEmpty(skipBuildPhrase)) {
                 logger.log(Level.INFO,
-                           "Pull request commented with {0} skipBuildPhrase. Hence skipping the build.",
-                           skipBuildPhrase);
+                        "Pull request commented with {0} skipBuildPhrase. Hence skipping the build.",
+                        skipBuildPhrase);
+                shouldRun = false;
+                return;
+            }
+            if (commitAuthor == null){
+                return;
+            }
+            String blackListCommitAuthor = helper.checkBlackListCommitAuthor(commitAuthor.getName());
+            if (!StringUtils.isEmpty(blackListCommitAuthor)) {
+                logger.log(Level.INFO,
+                        "Pull request triggered by {0} user. Hence skipping the build.",
```

Comment:
Would you mind rewording it? Here's my suggestion:
`Pull request triggered by {0} user. Skipping build because that user is blacklisted.`